logdevice/common/NodesConfigurationPoller.cpp

```diff
@@ -22,10 +22,12 @@ NodesConfigurationPoller::NodesConfigurationPoller(
     Poller::Options options,
     VersionExtFn version_fn,
     Callback cb,
+    folly::Optional<u_int32_t> node_order_seed,
     folly::Optional<Version> conditional_base_version)
     : options_(std::move(options)),
       version_fn_(std::move(version_fn)),
       cb_(std::move(cb)),
+      node_order_seed_(node_order_seed),
```

Comment:
The order of the fields in the constructor should match the order of the fields in the header.